PowerShellForGitHub.Format.ps1xml

```diff
@@ -0,0 +1,145 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<Configuration>
+  <ViewDefinitions>
+    <!--================== GitHub.Repository Type View =================-->
+    <View>
+      <Name>GitHub.Repository</Name>
+      <ViewSelectedBy>
+        <TypeName>GitHub.Repository</TypeName>
+      </ViewSelectedBy>
+      <TableControl>
+        <TableRowEntries>
+          <TableRowEntry>
+            <TableColumnItems>
+              <TableColumnItem>
+                <PropertyName>full_name</PropertyName>
+              </TableColumnItem>
+              <TableColumnItem>
+                <PropertyName>private</PropertyName>
```

Comment:
```suggestion
                <PropertyName>visibility</PropertyName>
```

It might be more readable to have this column just be "visibility" and then have a scriptblock for the TableColumnItem that prints 'public' or 'private' based on the state.

Reply:
I've just changed the column to `visibility`. Don't think there is any need for a script block?

Reply:
I think I wrote that comment either before I actually reviewed your other PR that introduced the `Visibility` property, or I simply forgot about it.  Correct...given that there is an actual `Visibility` property that uses the words we want, there's no need for a scriptblock to accomplish that task.